SEMVER.md

```diff
@@ -0,0 +1,39 @@
+# Semantic versioning
+
+LUX gebruikt Semantic versioning (SemVer)[^1] voor de versies van releases.
+Dit is een wijd gebruikt versienummeringssysteem, dat in de basis uit drie versienummers bestaat, namelijk:
+
+> MAJOR.MINOR.PATCH
+
+Deze kunnen uitgebreid worden met optionele pre-release- en build-metadata labels.
+
+De verschillende elementen worden, volgens het officiele schema[^2], verhoogd als volgt:
+
+> 1. MAJOR wordt verhoogd bij incompatibele API-wijzigingen,
+> 1. MINOR wordt verhoogd bij het toevoegen van functionaliteit die compatibel is met de vorige versie, en
+> 1. PATCH wordt verhoogd bij compatibele bugfixes.
+
+Binnen LUX hebben we dit uitgebreidt en worden de versienummer als volgt verhoogd:
+
+<dl>
+<dt>MAJOR</dt>
+<dd>wordt verhoogd bij incompatibele én impactvolle (visuele) wijzigingen die bij gebruikers van LUX toegankelijkheids- of regressieproblemen kunnen veroorzaken,</dd>
+<dt>MINOR</dt>
+<dd>wordt verhoogd bij het toevoegen van functionaliteit die compatibel is met de vorige versie, en</dd>
+<dt>PATCH</dt>
+<dd>wordt verhoogd wordt verhoogd bij compatibele bugfixes.</dd>
+</dl>
+
+Dus een wijziging in kleuren van een component die bij een design bijvoorbeeld tot contrastproblemen of grote (visuele) regressie kunnen veroorzaken geldt als een Major versie-update.  
+Het is niet uitgesloten dat een Minor versie ook (visuele) regressietesten breekt, maar hiervoor zouden geen wijzigingen in een design of andere code-aanpassingen voor nodig hoeven zijn, alleen een update van de test of accorderen van een nieuwe baseline.
+
+Op dit moment is dit de basis van onze versionering, in de loop van de tijd zullen we deze aanscherpen en aanvullen met voorbeelden uit de praktijk.
+
+Mochten er nog vragen zijn, neem contact op met het LUX-team via ons [Slack #support-kanaal](https://logius-ux.slack.com/archives/C072WMXCC3H).
```

Comment:
Goed dat je Slack nog even noemt. Uiteindelijk krijgen we een Contact/Support pagina waardoor dit niet meer nodig is. Voor nu is dit wel heel goed